src/SWP/Bundle/ContentBundle/Hydrator/ArticleHydrator.php

```diff
@@ -80,8 +72,7 @@ public function hydrate(ArticleInterface $article, PackageInterface $package): A
         $article->setLead($this->populateLead($package));
         $article->setMetadata($package->getMetadata());
         $article->setKeywords($package->getKeywords());
-        // assign default route
-        $article->setRoute($this->routeProvider->getRouteForArticle($article));
+        $article->setRoute($article->getRoute());
```

Comment:
why do you set already set route? I can be removed IMO.

Reply:
i replaced this what provider was doing (https://github.com/superdesk/web-publisher/pull/424/files#diff-0f154de2d5182f063d816e8fad2a9bf4L115). But without this dependency.